netbox/dcim/models/devices.py

```diff
@@ -1272,7 +1272,7 @@ class MACAddress(PrimaryModel):
     )
 
     class Meta:
-        ordering = ('mac_address',)
+        ordering = ('mac_address', 'id',)
```

Comment:
```suggestion
        ordering = ('mac_address', 'pk',)
```

I believe we use `pk` instead of `id` in all other locations where we have done this.